tests/middleware/test_exception_handler_middleware.py

```diff
@@ -118,3 +118,30 @@ async def after_exception_hook_handler(exc: Exception, scope: "Scope", state: "S
         response = client.get("/test")
         assert response.status_code == HTTP_500_INTERNAL_SERVER_ERROR
         assert client.app.state.called
+
+
+def test_exception_handler_middleware_debug_logging() -> None:
+    from unittest.mock import patch
+
+    @get("/test")
+    def handler() -> None:
+        raise RuntimeError()
+
+    with create_test_client(handler) as client, patch("logging.Logger.debug") as mock_debug:
```

Comment:
to enable logging you should pass a `LoggingConfig` to the app - you can do this with the `create_test_client` helper as well, simply pass `logging_config=LoggingConfig()`.